upx.yaml

```diff
@@ -0,0 +1,61 @@
+package:
+  name: upx
+  version: 4.0.2
+  epoch: 0
+  description: "The Ultimate Packer for eXecutables"
+  copyright:
+    - license: GPL-2.0
+
+environment:
+  contents:
+    # repositories:
+    #   - https://packages.wolfi.dev/os
+    # keyring:
+    #   - https://packages.wolfi.dev/os/wolfi-signing.rsa.pub
```

Comment:
We can remove these comments, when building locally we can run
```
USE_CACHE=no MELANGE=melange make packages/upx  BUILDWORLD=no
```
which adds the repository and keyring using CLI flags